airflow/bin/airflow

```diff
@@ -28,5 +28,6 @@ if __name__ == '__main__':
         os.environ['KRB5_KTNAME'] = configuration.conf.get('kerberos', 'keytab')
 
     parser = CLIFactory.get_parser()
-    args = parser.parse_args()
-    args.func(args)
+    args = parser.parse_known_args()
+    args[0].__setattr__('gunicorn_config', args[1])
```

Comment:
Do you need to use `__setattr__` here? would `args[0].gunicorn_config = args[1]` not do the same, or is the Namespace object being difficult on us?

Reply:
@ashb, you right, it's not needed - works fine without it. Changed.